src/rabbit_amqqueue.erl

```diff
@@ -586,6 +592,14 @@ info_all(VHostPath, Items) ->
     map(list(VHostPath), fun (Q) -> info(Q, Items) end) ++
         map(list_down(VHostPath), fun (Q) -> info_down(Q, Items, down) end).
 
+info_all(VHostPath, Items, Ref, AggregatorPid) ->
```

Comment:
info_all/2 calls `map/2` which in turns calls `rabbit_misc:filter_exit_map/2`. Why is your code not replicating that behaviour?